### PR TITLE
add poliastro

### PR DESCRIPTION
**Pull request by epassaro**
This is a continuation of #25 @esc @astrojuanlu

I tried to avoid maintaining dependencies on another file (enough with `pyproject.toml` and `conda-forge` recipe) by:

- Installing `poliastro` from `conda-forge` (all runtime packages are pulled from `conda-forge`)
- Re-installing `poliastro` package from source + `dev` dependencies with `pip install -e .[dev]`

Results:  `938 passed, 59 skipped, 33 deselected, 5 xfailed, 1 warning in 136.64s (0:02:16)`

### CODE REVIEW

**Comment by esc** [comment]
@epassaro thank you, this looks good. I would be worried about having multiple copies of poliastro installed. You are placing a lot of trust in both `pip` and `conda` and even more, them working together well. I can merge this, however if you feel confident enough. Let me know.

**Comment by epassaro** [comment]
> @epassaro thank you, this looks good. I would be worried about having multiple copies of poliastro installed. You are placing a lot of trust in both `pip` and `conda` and even more, them working together well. I can merge this, however if you feel confident enough. Let me know.

Let see what @astrojuanlu says.

In a previous commit, I force removed `poliastro` by doing `conda remove --force poliastro`, but the result was exactly the same (for now).  Maybe that's a good extra safety measure to not have multiple versions installed.

**Comment by astrojuanlu** [comment]
Thanks a lot @epassaro ! 🙌🏽  The test results look good.

Since we are not declaring extra dependencies in the conda recipe, doing this is probably the simplest way of getting the necessary pytest plugins. Options:

1. Merge this as is. But I agree with @esc that installing poliastro itself both with pip and with conda is worrying, and perhaps brittle.
2. Remove poliastro from conda before installing it with pip, as @epassaro suggested. I can't see any disadvantages.
2. Repeat dev dependencies somewhere here, and install poliastro only once either with conda or with pip.

I wonder if there are any best practices that other projects follow that we should imitate.

**Comment by esc** [comment]
> Thanks a lot @epassaro ! 🙌🏽 The test results look good.
> 
> Since we are not declaring extra dependencies in the conda recipe, doing this is probably the simplest way of getting the necessary pytest plugins. Options:
> 
> 1. Merge this as is. But I agree with @esc that installing poliastro itself both with pip and with conda is worrying, and perhaps brittle.
> 2. Remove poliastro from conda before installing it with pip, as @epassaro suggested. I can't see any disadvantages.
> 3. Repeat dev dependencies somewhere here, and install poliastro only once either with conda or with pip.
> 
> I wonder if there are any best practices that other projects follow that we should imitate.

It's a but silly in terms of duplication, but I have found 3. to be the most useful. If projects change dependencies often, that is a problem...

**Comment by astrojuanlu** [comment]
Fair enough - we don't change dev dependencies often. Here they are:

https://github.com/poliastro/poliastro/blob/1b3da82d82328c5d748959455d93b18b1e740364/pyproject.toml#L70-L98

However, I do realize that some of these dependencies are actually for the documentation. In the spirit of not blocking this PR any longer before I can sit and sort them, I _think_ this subset should be enough:

```
     "coverage", 
     "hypothesis", 
     "mypy>=0.740", 
     "pytest>=3.2", 
     "pytest-cov<2.6.0",  # https://github.com/pytest-dev/pytest-cov/issues/222 
     "pytest-doctestplus>=0.8",  # https://github.com/astropy/pytest-doctestplus/issues/118 
     "pytest-mpl", 
     "pytest-mypy", 
     "pytest-remotedata", 
```

I might be missing something, but not much.

**Review by astrojuanlu — APPROVED** [review]
Looks great to me as long as the tests pass! Thanks again @epassaro 🙌🏽

**Comment by epassaro** [comment]
> Looks great to me as long as the tests pass! Thanks again @epassaro 🙌🏽

Ok, the tests passed. I just squashed the commits right now.

**Comment by epassaro** [comment]
This is ready to merge @esc

**Comment by esc** [comment]
> This is ready to merge @esc

Awesome thank you!

**Pull request merged**
